yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -155,6 +155,29 @@ func saveMigrationReportsFn(msr *metadb.MigrationStatusRecord) {
 		}
 	}
 
+	askAndStorePasswords(msr)
+	passwordsEnvVars := []string{
+		fmt.Sprintf("TARGET_DB_PASSWORD=%s", targetDBPassword),
+		fmt.Sprintf("FF_DB_PASSWORD=%s", fallForwardDBPassword),
+		fmt.Sprintf("SOURCE_DB_PASSWORD=%s", sourceDBPassword),
+	}
+
+	if msr.ExportType == "snapshot-and-changes" { // streaming changes case
+		utils.PrintAndLog("save live migration reports...")
+		liveMigrationReportFilePath := filepath.Join(backupDir, "reports", "live_migration_report.txt")
+		strCmd := fmt.Sprintf("yb-voyager live-migration report -e %s > %q", exportDir, liveMigrationReportFilePath)
```

Comment:
You'll have to change the command based on https://github.com/yugabyte/yb-voyager/pull/1194. 

Take care of this while merging!